hub/core/transform/transform.py

```diff
@@ -237,9 +236,8 @@ def _run(progress_port=None):
             _run()
 
         if overwrite:
-            chunk_paths = get_chunk_paths(target_ds, tensors)
-            # TODO:
-            # delete_chunks(chunk_paths, storage, compute)
+            for tensor in target_ds.tensors.values():
```

Comment:
could you add a test within one of the inplace transforms that confirms that the chunks were deleted? You can test using local storage provider that will be able to list all the remaining chunks in the storage after deletion.